templates/service_review_badge.php

```diff
@@ -0,0 +1,14 @@
+<?php
+  if ( ! defined( 'ABSPATH' ) ) {
+    exit; // Exit if accessed directly
+  }
+
+  $widget_width = isset( $this->width ) ? $this->width : "50px";
+  $widget_height = isset( $this->height ) ? $this->height : "50px";
+  $custom_classes = isset( $this->class ) ? $this->class : "";
+?>
+
+<div class="lipscore-service-review-badge-starred <?php echo esc_attr( $custom_classes ); ?>"
+    ls-widget-height="<?php echo esc_attr( $widget_height ); ?>"
+    ls-widget-width="<?php echo esc_attr( $widget_width ); ?>">
+</div>
```

Comment:
> Why do we need `custom classes` for the service review badge and testimonial banner widgets?

Sometimes our clients want to change the color of the badge or make any other styling changes. I thought it would be a good idea to have a way to pass custom CSS classes for these widget's container elements, making them more customizable via CSS.